karoo_gp/base_class.py

```diff
@@ -1787,1535 +2043,1588 @@ class labels provided in the data .csv. Outputs an array of tuples
-        through bottom, within the given tree.
 
-        Called by: fx_karoo_gp, fx_karoo_pause
+#+++++++++++++++++++++++++++++++++++++++++++++
+#   Methods to Visualize a Tree              |
+#+++++++++++++++++++++++++++++++++++++++++++++
+# used by: Tree
+def fx_display_tree(tree):
 
-        Arguments required: tree
-        '''
+    '''
+    Display all or part of a Tree on-screen.
 
-        ind = ''
-        print('\n\033[1m\033[36m Tree ID', int(tree[0][1]), '\033[0;0m')
-
-        # increment through all possible Tree depths - tested 2016 07/09
-        for depth in range(0, self.tree_depth_max + 1):
-            print('\n', ind, '\033[36m Tree Depth:', depth, 'of',
-                  tree[2][1], '\033[0;0m')
-
-            # increment through all nodes (redundant, I know)
-            for node in range(1, len(tree[3])):
-                if int(tree[4][node]) == depth:
-                    print('')
-                    print(ind, '\033[1m\033[36m NODE:', tree[3][node],
-                          '\033[0;0m')
-                    print(ind, '  type:', tree[5][node])
-                    print(ind, '  label:', tree[6][node], '\tparent node:',
-                          tree[7][node])
-                    print(ind, '  arity:', tree[8][node], '\tchild node(s):',
-                          tree[9][node], tree[10][node], tree[11][node])
-
-            ind = ind + '\t'
-
-        print('')
-        # generate the raw and sympified expression for the entire Tree
-        self.fx_eval_poly(tree)
-        print('\t\033[36mTree', tree[0][1], 'yields (raw):',
-              self.algo_raw, '\033[0;0m')
-        print('\t\033[36mTree', tree[0][1], 'yields (sym):\033[1m',
-              self.algo_sym, '\033[0;0m')
+    This method displays all sequential node_ids from 'start' node
+    through bottom, within the given tree.
 
-        return
+    Called by: fx_karoo_gp, fx_karoo_pause
 
+    Arguments required: tree
+    '''
 
-    def fx_display_branch(self, tree, start):
+    ind = ''
+    print('\n\033[1m\033[36m Tree ID', int(tree.root[0][1]), '\033[0;0m')
+
+    # increment through all possible Tree depths - tested 2016 07/09
+    for depth in range(0, tree.tree_depth_max + 1):
+        print('\n', ind, '\033[36m Tree Depth:', depth, 'of',
+                tree.root[2][1], '\033[0;0m')
+
+        # increment through all nodes (redundant, I know)
+        for node in range(1, len(tree.root[3])):
+            if int(tree.root[4][node]) == depth:
+                print('')
+                print(ind, '\033[1m\033[36m NODE:', tree.root[3][node],
+                        '\033[0;0m')
+                print(ind, '  type:', tree.root[5][node])
+                print(ind, '  label:', tree.root[6][node], '\tparent node:',
+                        tree.root[7][node])
+                print(ind, '  arity:', tree.root[8][node], '\tchild node(s):',
+                        tree.root[9][node], tree.root[10][node], tree.root[11][node])
```

Comment:
Should these be `log`?
```suggestion
                print()
                print(ind, '\033[1m\033[36m NODE:', tree.root[3][node],
                      '\033[0;0m')
                print(ind, '  type:', tree.root[5][node])
                print(ind, '  label:', tree.root[6][node], '\tparent node:',
                      tree.root[7][node])
                print(ind, '  arity:', tree.root[8][node], '\tchild node(s):',
                      tree.root[9][node], tree.root[10][node], tree.root[11][node])
```